### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #851 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#426]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 8.99 KB...]
database.put(
    "UserProfile",
    {"numOfReviews" = 5, "rating" = 25, "profileImg" = org.parse4j.ParseFile@40f538fa, "username" = "alexaxa"}
);
-> at smartcity.accessibility.database.UserProfileManager.lambda$put$1(UserProfileManager.java:131)

	at smartcity.accessibility.database.UserProfileManagerTest.testPut(UserProfileManagerTest.java:80)

Running smartcity.accessibility.search.SearchTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.558 sec - in smartcity.accessibility.search.SearchTest

Results :

Failed tests: 
  UserProfileManagerTest.testPut:80 
Argument(s) are different! Wanted:
database.put(
    "UserProfile",
    {"numOfReviews" = 5, "rating" = 25, "profileImg" = null, "username" = "alexaxa"}
);
-> at smartcity.accessibility.database.UserProfileManagerTest.testPut(UserProfileManagerTest.java:80)
Actual invocation has different arguments:
database.put(
    "UserProfile",
    {"numOfReviews" = 5, "rating" = 25, "profileImg" = org.parse4j.ParseFile@40f538fa, "username" = "alexaxa"}
);
-> at smartcity.accessibility.database.UserProfileManager.lambda$put$1(UserProfileManager.java:131)


Tests run: 77, Failures: 1, Errors: 0, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:24 min
[INFO] Finished at: 2017-06-09T10:33:20+00:00
[INFO] Final Memory: 34M/218M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [yaelAmitay] f7d8ddfdd79e727a6497da330ce2e79ea7fddc8e - add loading spinner to sign up

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/851/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
